azure/README.md

```diff
@@ -119,21 +97,24 @@ Use the `ManagedImageSharedImageGalleryId` field from output of the above comman
 export AZURE_IMAGE_ID="REPLACE_ME"
 ```
 
+### Option 2: Customize an Image Using Prebuilt Binaries via Docker
 
-You can also build the image using docker
 ```bash
 cd image
-docker build -t azure \
-  --secret id=AZURE_CLIENT_ID \
-  --secret id=AZURE_CLIENT_SECRET \
-  --secret id=AZURE_SUBSCRIPTION_ID \
-  --secret id=AZURE_TENANT_ID \
-  --build-arg AZURE_RESOURCE_GROUP=${AZURE_RESOURCE_GROUP} \
-  -f Dockerfile .
+docker build -t azure-podvm-builder .
 ```
 
-If you want to use a different base image, then you'll need to provide additional build-args:
-`PUBLISHER`, `OFFER`, `SKU`
+```bash
+docker run --rm \
+  -v "$HOME/.azure:/root/.azure" \
```

Comment:
I think this will break the current workflow of creating images from within a Kubernetes cluster. This is fine as long as there is some way to perform non-interactive container based podvm image builds.

So with the change in this PR, if I perform a non-interactive `az login` with subscription, tenant, client secret before running the image build would that work ? 
For me non-interactive login didn't work since it always asks me to the open a web browser and enter the device code.

Reply:
> So with the change in this PR, if I perform a non-interactive `az login` with subscription, tenant, client secret before running the image build would that work ? For me non-interactive login didn't work since it always asks me to the open a web browser and enter the device code.

So I did have two scenarios in mind:
- Locally triggered builds, using the developer's az credentials (probably via interactive `az login` auth`)
- CI: Federated credentials w/ OIDC via an `azure/login@v1` github action.

Are there any other scenarios we should cover? Can you point me what you are referring to?

> I think this will break the current workflow of creating images from within a Kubernetes cluster.

Reply:
@mkulke We use a K8s job to create the image https://github.com/openshift/sandboxed-containers-operator/blob/devel/hack/azure-image-job.yaml
But this PR doesn't change anything w.r.to packer config. So I don't see any problems with running the K8s job.
I also tried the dev workflow after login and it worked.